README.md

```diff
@@ -9,23 +9,40 @@ cycle.
 
 # Install and run the website locally (for development)
 
-_NB: The following instructions are for Linux.
-On Mac, Ruby version 2.6 may be necessary. You can install a specific version
-with [rbenv](https://github.com/rbenv/rbenv)._
-
 1. Clone the repository, move to its directory
     ```
     git clone git@github.com:eWaterCycle/ewatercycle.github.io.git
     cd ewatercycle.github.io
     ```
+1. Run with docker (recommended) or install Ruby and run locally
+
+## Install and run with Docker
+
+1. Ensure Docker is running.
+
+1. Run from the command line:
+```
+docker run --rm -it \
+  --volume="$PWD:/srv/jekyll" \
+  --volume="$PWD/vendor/bundle:/usr/local/bundle" \
+  -p 4000:4000 jekyll/jekyll:3.8 \
+  jekyll serve
```

Comment:
This command doesn't work as expected for me: it builds and serves, but I cannot reach the address. Maybe this is due to my local settings, I'd have to delve into it to find out why.

Reply:
The difference with the local build is that this command serves at http://0.0.0.0:4000 whereas the local build serves at https://127.0.0.1:4000

Reply:
Thanks a lot for trying this out!

Could you try if this works:

```
docker run --rm -it \
  --volume="$PWD:/srv/jekyll" \
  --volume="$PWD/vendor/bundle:/usr/local/bundle" \
  -p 127.0.0.1:4000:4000 \
  jekyll/jekyll:3.8 \
  jekyll serve
```

If that doesn't work either, here is the docker command that @sverhoeven used:
```
docker run --rm -it \
  --volume=$(pwd):/srv/jekyll \
  -p 127.0.0.1:4000:4000 \
  jekyll/jekyll:pages \
  jekyll serve
```
does this work?

Reply:
Unfortunately, they don't work either. Also the final server address is still 0.0.0.0:4000. Could be due to my local setup that this doesn't work, though..